examples/dial/example_dial.cc

```diff
@@ -22,14 +23,14 @@ extern "C" char *dial(const char *uri, const char *payload, bool allow_insecure,
 		      void *ptr);
 
 int main() {
-	const char *uri = "<your robot URI here>";
+	const char *uri = "naveed-pi-main.60758fe0f6.viam.cloud";
```

Comment:
Just wondering, why are you including an actual robot URI here? Just to further flesh out the `example`?

Also, does it matter that it is naveed-**pi**-main...? I'm assuming its just a URI from a python robot you made or something? Thanks!